README.md

```diff
@@ -152,6 +156,37 @@ func main() {
 }
 ```
 
+## SAMS REST API
+
+The SAMS REST API is for user-oriented access tokens. These APIs are much simpler in nature, as most integrations
+will make use of the Client API. However, the REST API is required if the service is not governing access based
+on the SAMS token scope, but instead using its own authorization mechanism. e.g. governing access based on the
+SAMS external account ID.
+
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	samsREST "github.com/sourcegraph/sourcegraph-accounts-sdk-go/rest"
```

Comment:
Convention here is to use all lowercase letters 😁 
```suggestion
	samsrest "github.com/sourcegraph/sourcegraph-accounts-sdk-go/rest"
```

Reply:
Done